packages/flutter/lib/src/material/slider.dart

```diff
@@ -684,26 +684,41 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
   void _actionHandler(_AdjustSliderIntent intent) {
     final _RenderSlider renderSlider = _renderObjectKey.currentContext!.findRenderObject()! as _RenderSlider;
     final TextDirection textDirection = Directionality.of(_renderObjectKey.currentContext!);
+    bool sliderIncreased = false;
+    widget.onChangeStart?.call(widget.value);
+
     switch (intent.type) {
```

Comment:
Question:  When a case statement doesn't end with a break, most C-style languages either fall through or error at compile time.  From my reading of this https://dart.dev/language/control-flow#switch-and-case - I'd expect a compiler error here.  And if it falls through, I'd expect some really weird behavior.  Even through you didn't introduce this, I'd like to understand what is going on here as a Dart newbie.

Reply:
I asked on Discord and got this link and response: "This used to be a syntax error, but the requirement was thrown out when pattern matching was added (it behaves the same as if there was a break statement, unlike C-style switches)"
link-> https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#language-1

Reply:
Thanks Devin!  That makes total sense - and it looks like the 3.0 release is new enough that the rest of the documentation hasn't caught up.